Worker.Extension.Sql/src/SqlInputAttribute.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extension.Sql
+{
+    public sealed class SqlInputAttribute : InputBindingAttribute
+    {
+        /// <summary>
+        /// Constructs a new instance.
```

Comment:
This isn't a super helpful description - it should describe what the class is actually doing